src/components/staking/PageValidator.vue

```diff
@@ -107,11 +107,11 @@
             {{ (validator.selfStake / validator.tokens) | percent }}
           </span>
         </li>
-        <li>
+        <li v-if="validator.startHeight">
```

Comment:
i'm assuming that livepeer doesn't have this data? 

is there no way to find out when a transcoder came online / became a transcoder? 

if no, let's say "unknown" or "n/a" or "--" instead of hiding the div.

Reply:
same for all the changes below in this file.

Reply:
Ok :+1: